index.d.ts

```diff
@@ -24,8 +24,15 @@ declare namespace authService {
   function setFormsKeyToken(formsKeyToken: string): void
   function isAuthorised(formsAppId: number): Promise<boolean>
   function requestAccess(formsAppId: number): Promise<void>
-  function login(): Promise<string>
+  function loginHostedUI(identityProvider: string | void): Promise<string>
   function handleAuthentication(): Promise<string>
+  function loginUsernamePassword(): Promise<
+    ((newPassword: string) => Promise<void>) | void
+  >
+  function changePassword(): Promise<void>
```

Comment:
This should new and old passwords right?

Reply:
yep